gulpfile.js

```diff
@@ -174,7 +174,7 @@ async function updateBuildNumber(args) {
             versionParts.length > 2 ? versionParts[2].replace(/(\d+)/, args.buildNumber) : args.buildNumber;
         const newVersion =
             versionParts.length > 1
-                ? `${versionParts[0]}.${versionParts[1]}.${buildNumberPortion}`
+                ? `${versionParts[0]}.${versionParts[1]}.9${buildNumberPortion.substring(0, buildNumberPortion.length - 1)}`
```

Comment:
Same version issue, this will make this version be the highest on stable.

Reply:
We don't need this.